src/main/java/ui/listpanel/ListPanelCell.java

```diff
@@ -20,6 +20,8 @@
     private final ListPanel parent;
     private final HashSet<Integer> issuesWithNewComments;
     private TurboIssue issue;
+    private final String closedStyle = "issue-cell-closed";
+    private boolean isCurrentStyleClosed = getStyleClass().contains(closedStyle);
```

Comment:
rather, I meant the two variables should be declared inside the method.  It's safer to keep the scope of variables as narrow as possible.